### PR TITLE
update version of hapi to 12.1.0 (latest) 

### LINKED ISSUE #79: Update Copy to Focus less on MVP
Can we do a quick update to *expand* our service to more full-featured "digital products" instead of "Prototypes"..?

### PR DESCRIPTION
**Pull request by nelsonic**
## Jan 2016 - Before Huw Applies His Magic... :wink: 

+ update copy to be broader (less MVP-focussed fixes #79)
+ fixes https://github.com/dwyl/dwyl.github.io/issues/81

### CODE REVIEW

**Comment by nelsonic** [comment]
@iteles can you please review the *copy* changes. thanks. :+1:

File: index.html [hunk 1]
```diff
@@ -119,24 +116,33 @@ <h2 class="content-head content-head-ribbon">What do we do?</h2>
...
+        to getting idea(s) tested in the hands of <em>real</em> people.
       </p>
       <ol>
         <li>
-          <i>Understand</i> the problem that needs to be solved
-          and confirm the intended users feel the <i>pain</i>
-          and <i>need</i> for a solution.
+          <em>Understand</em> the problem that has been identified
+          and <em>confirm</em> the intended users feel the <i>pain</i>
+          and <em>need</em> for a solution.
```
Comment (iteles) [inline index.html]: Semantically, I think this should be an `<i>` instead of an `<em>` but it's not really worth spending that much time debating! :blush:

**Comment by iteles** [comment]
:+1: Great start, I'll spend more time on this over the weekend.

File: index.html (hunk 1 above)
Comment (nelsonic) [inline index.html]: Actually, I agree. please `find-and-replace` when you do your edits. thanks.

**Comment by iteles** [comment]
Seems like my latest commit didn't get updated into the website... odd.

**Comment by nelsonic** [comment]
Heroku requires *manual* deploy ...
re-deploy-dwyl-website-to-heroku

Thanks for fixing the typos. :+1:

**Pull request merged**